src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -152,7 +152,16 @@ public void CounterPayloadReceived(string providerName, ICounterPayload payload,
 
         public void Stop()
         {
-            // Nothing to do here.
+            if (initialized)
+            {
+                var row = providers.Values.SelectMany(c => c.Counters.Values).Select(c => c.Row).DefaultIfEmpty(-1).Max();
```

Comment:
I don't know what's the threading model for the application, so I'm not sure it's always safe to access `providers` here. If not, we can either add a lock or a `_maxRow` field that would be updated by the thread calling `CounterPayloadReceived`